lib/components/user/stacked-pane-display.js

```diff
@@ -1,36 +1,40 @@
 import PropTypes from 'prop-types'
-import React from 'react'
+import React, {useState} from 'react'
 
 import FormNavigationButtons from './form-navigation-buttons'
 import { PageHeading, StackedPaneContainer } from './styled'
 
 /**
  * This component handles the flow between screens for new OTP user accounts.
  */
-const StackedPaneDisplay = ({ onCancel, paneSequence, title }) => (
-  <>
-    {title && <PageHeading>{title}</PageHeading>}
-    {
-      paneSequence.map(({ pane: Pane, props, title }, index) => (
-        <StackedPaneContainer key={index}>
-          <h3>{title}</h3>
-          <div><Pane {...props} /></div>
-        </StackedPaneContainer>
-      ))
-    }
+const StackedPaneDisplay = ({ onCancel, paneSequence, title }) => {
+  // Create indicator of if cancel button was clicked so that child components can know
+  const [isBeingCanceled, updateBeingCanceled] = useState(false)
 
-    <FormNavigationButtons
-      backButton={{
-        onClick: onCancel,
-        text: 'Cancel'
-      }}
-      okayButton={{
-        text: 'Save Preferences',
-        type: 'submit'
-      }}
-    />
-  </>
-)
+  return (
+    <>
+      {title && <PageHeading>{title}</PageHeading>}
+      {
+        paneSequence.map(({ pane: Pane, props, title }, index) => (
+          <StackedPaneContainer key={index}>
+            <h3>{title}</h3>
+            <div><Pane canceled={isBeingCanceled} {...props} /></div>
+          </StackedPaneContainer>
+        ))
+      }
+
+      <FormNavigationButtons
+        backButton={{
+          onClick: () => { updateBeingCanceled(true); onCancel() },
```

Comment:
What formatting do @landonreed and @evansiroky prefer regarding the semicolon? Spread over multiple lines?

Reply:
This was created by prettier -- ambiguities like this should be resolved in the future once we get the new version of mastarm up and running (which should include auto formatting).

Reply:
Yea, I was stuck on this a bit also, @binh-dam-ibigroup. Let's just go ahead and move to multiple lines, since we tend not to include semicolons anywhere in this code base.